odd_collector_azure/adapters/azure_sql/adapter.py

```diff
@@ -0,0 +1,45 @@
+import logging
```

Comment:
Create named logger in `logger.py` for azuresql, and use it instead of logging, this way we can separate messages from adapters

```python
import logging

logger = logging.getLogger('azure')
```

Reply:
Created.